### DISCUSSION TITLE
new car , new tesla account

### DISCUSSION THREAD

**Discussion by merlinfive** [teslamate-org/teslamate#3165] [Q&A] [answered]
Hi ,
I have teslamate running fine with my account and car right now ,
in a few weeks I get a new Tesla which I ordered with new email address .
How do I get rid of the old one and add the new one ?

**Comment by CarlosCuezva** [comment, marked as answer]
Hello.

You can follow the steps to restore a backup, specifically the first 2 steps and the last one

1.- # Stop the teslamate container to avoid write conflicts
2.- # Drop existing data and reinitialize
3.- # Restart the teslamate container

https://docs.teslamate.org/docs/maintenance/backup_restore#restore

Now, you will have a clean instalation of Teslamate

**Reply by merlinfive** [reply]
perfect , thanks will try it this way in a few weeks , tried in my "sandbox" works like charm
